### ISSUE TITLE
Document project hooks

### ISSUE THREAD

**Issue by ethagnawl** [tmuxinator/tmuxinator#558]
Document project hooks introduced by #486.

This feature is currently outlined in sample.yml, but it would be nice to also have an entry in README.md.

**Comment by HaleTom** [comment]
I came here because I was getting the error:
```
    DEPRECATION: the post option has been replaced by project hooks and will
    not be supported anymore.
```

Here is the [`sample.yml`](https://github.com/tmuxinator/tmuxinator/pull/486/files#diff-09f3f2ccaa37af5fe72b53622f0e356f) referred to above.

I replaced my `post:` with:
```
on_project_first_start: tmux --<command-line-options>
```

**Comment by weijiekoh** [comment]
#576 Hope this pull request can address this issue.

**Comment by ethagnawl** [comment]
Closed via #576.
